Modules/ThirdParty/Eigen3/UpdateFromUpstream.sh

```diff
@@ -7,12 +7,10 @@ shopt -s dotglob
 readonly name="Eigen3"
 readonly ownership="Eigen Upstream <kwrobot@kitware.com>"
 readonly subtree="Modules/ThirdParty/Eigen3/src/itkeigen"
-# readonly repo="https://github.com/eigenteam/eigen-git-mirror"
```

Comment:
Leaving the official mirror here will probably save somebody some searching.

Reply:
That's was the non-official mirror when eigen was in mercurial. Now they have a proper git version here:
https://gitlab.com/libeigen/eigen.

I will put it there.